9.2/run-postgresql.sh

```diff
@@ -90,12 +107,16 @@ function initialize_database() {
 
 		# Allow connections from all hosts.
 		host all all all md5
+
+		# Allow replication connections from all hosts.
+		host replication all all md5
 	EOF
 
 	pg_ctl -w start
 	createuser "$POSTGRESQL_USER"
 	createdb --owner="$POSTGRESQL_USER" "$POSTGRESQL_DATABASE"
 	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH ENCRYPTED PASSWORD '${POSTGRESQL_PASSWORD}';"
+	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH REPLICATION ;"
```

Comment:
My paas thinking is still in early development, but in container we treat `POSTGRESQL_USER` as kind of admin, right? So having `REPLICATION` role set for this user doesn't seem too risky. It shouldn't be set if it is not necessary though, as proposed by @mnagy.